Source/Scene/ParticleSystem.js

```diff
@@ -0,0 +1,295 @@
+/*global define*/
+define([
+        '../Core/defaultValue',
+        '../Core/defined',
+        '../Core/Cartesian2',
+        '../Core/Cartesian3',
+        '../Core/Event',
+        '../Core/Matrix4',
+        '../Core/Math',
+        '../Core/JulianDate',
+        '../Core/Color',
+        './BillboardCollection',
+        './Particle',
+        './CircleEmitter'
+    ], function(
+        defaultValue,
+        defined,
+        Cartesian2,
+        Cartesian3,
+        Event,
+        Matrix4,
+        CesiumMath,
+        JulianDate,
+        Color,
+        BillboardCollection,
+        Particle,
+        CircleEmitter) {
+    "use strict";
+
+    var ParticleSystem = function(options) {
+        options = defaultValue(options, defaultValue.EMPTY_OBJECT);
+
+        this.particles = defaultValue(options.particles, []);
+        this.forces = defaultValue(options.forces, []);
```

Comment:
Should all these properties be get/set?

Likewise, what properties should have a `DeveloperError` thrown for clearly wrong values like negative scale?